Microsoft Teams notifications documentation (#19)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/notification-configurations.mdx

The notification is an HTTP POST request with a detailed payload describing the run. The payload depends on the type of notification configured.

> For Slack and Microsoft Teams notifications, the payload conforms to the respective webhook API, and will result in a message posted with informational attachments. (For an example, see [the Slack section](/cloud-docs/workspaces/settings/notifications#slack) and [the Microsoft Teams section](/cloud-docs/workspaces/settings/notifications#microsoft-teams) of the notifications docs.) For Generic notifications, the payload has several fields:

Reviewer: ```suggestion
For Slack and Microsoft Teams notifications, the payload conforms to the respective webhook API, and results in a notification message attached information. Refer to [Slack Notification Payloads](/cloud-docs/workspaces/settings/notifications#slack) and [Microsoft Teams Notification Payloads](/cloud-docs/workspaces/settings/notifications#microsoft-teams)  for examples. For generic notifications, the payload has several fields:
```

Reviewer 2: There were two suggestions for this line, and I have applied the other one because `and results in a notification message attached information` is not clear to me. @laurapacilio I think that this line is good now, but correct me if I am missing something!

> For Slack and Microsoft Teams notifications, the payload conforms to the respective webhook API, and will result in a message posted with informational attachments. (For an example, see [the Slack section](/cloud-docs/workspaces/settings/notifications#slack) and [the Microsoft Teams section](/cloud-docs/workspaces/settings/notifications#microsoft-teams) of the notifications docs.) For Generic notifications, the payload has several fields:

Reviewer: ```suggestion
For Slack and Microsoft Teams notifications, the payload conforms to the respective webhook API, and results in a notification message with informational attachments. Refer to [Slack Notification Payloads](/cloud-docs/workspaces/settings/notifications#slack) and [Microsoft Teams Notification Payloads](/cloud-docs/workspaces/settings/notifications#microsoft-teams)  for examples. For generic notifications, the payload has several fields:
```
